ci-operator/config/openshift/cincinnati-graph-data/openshift-cincinnati-graph-data-master.yaml

```diff
@@ -48,12 +48,12 @@ raw_steps:
 releases:
   initial:
     integration:
-      name: "4.11"
+      name: "4.10"
```

Comment:
I don't understand this shift from 4.11 to 4.10.  Can you explain your motivation?

Reply:
the cluster pools dont have a 4.11 cluster as per a previous job run. so decided to move it to 4.10
here's the job https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/26913/rehearse-26913-pull-ci-openshift-cincinnati-graph-data-master-e2e-latest-cincinnati/1502035638516453376

Reply:
makes sense.  Can we add that motivation to the commit message?